packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -297,8 +297,9 @@ export function willForceUpload(
                 );
                 if (uploadResponse.success) {
                     Gui.showMessage(uploadResponse.commandResponse);
-                    if (node && uploadResponse?.apiResponse[0]?.etag) {
-                        node.setEtag(uploadResponse.apiResponse[0].etag);
+                    const apiResponse = uploadResponse.apiResponse ? uploadResponse.apiResponse[0] : {};
+                    if (node && apiResponse.etag) {
+                        node.setEtag(apiResponse.etag);
```

Comment:
I see this portion missing some code coverage, should we write a test for it or is this fine since it seemed to be passing before the removal of optional chaining? @traeok

Reply:
I see, so i'm assuming once that is merged, it should also fix your breaking check for codecov?

Reply:
Yep, I think we can let this PR sit until #2564 is merged and `main` is updated with the latest changes from `maintenance`. Once the changes are on main, I'll pick his incoming changes during the merge commit and that should resolve the patch coverage 🙂